pytest_opentmi/OpenTmiReport.py

```diff
@@ -230,7 +230,7 @@ def _new_result(self, report):
             elif key == 'SUT_BRANCH':
                 result.execution.sut.branch = value
             elif key == 'SUT_TAG':
-                result.execution.sut.tag = value
+                result.execution.sut.tag = [value]
```

Comment:
tag is list of strings (https://github.com/OpenTMI/opentmi-pyclient/blob/034c539d36fe13a2d6538ea421e4c01f00f5687d/test/test_api_result_sut.py#L27)